src/crud.js

```diff
@@ -0,0 +1,53 @@
+function editTaskEventListener(parentCheckBoxContainer) {
+  parentCheckBoxContainer.addEventListener('input', (event) => {
+    const { target } = event;
+    if (target.classList.value.indexOf('editable') !== -1) {
+      const tasksArray = JSON.parse(localStorage.getItem('tasksList'));
+      const inputWithId = target.previousElementSibling;
+      // eslint-disable-next-line max-len
+      const index = tasksArray.findIndex((element) => element.index === parseInt(inputWithId.id, 10));
+      tasksArray[index].description = target.innerHTML;
+      localStorage.setItem('tasksList', JSON.stringify(tasksArray));
+    }
+  });
+}
+
+const representNewIndexes = (tasksArray) => {
+  // eslint-disable-next-line no-plusplus
+  for (let i = 0; i < tasksArray.length; i++) {
```

Comment:
- [ ] Let us fix eslint errors without disabling them, this makes your code looks unprofessional.